settings.gradle.kts

```diff
@@ -37,8 +37,8 @@ include(
     "logging",
```

Comment:
BTW, could you please re-check that the artifact name for all of these modules (only those which are published!), is preprended with `spine-` prefix? We need it to be `<groupID>:spine-<artifactName>`, as we do in all similar cases.